tools/apigee-envoy-quickstart/README.md

```diff
@@ -0,0 +1,150 @@
+# Apigee Envoy Quickstart Toolkit
+
+The Apigee Envoy Quickstart Toolkit sets up the Envoy proxies with Apigee adapters. 
+<br/><br/>
+## Quickstart 
+<br/>
+
+[Apigee protected ASM Envoy Proxies within Kubernetes Engine](https://cloud.google.com/apigee/docs/api-platform/envoy-adapter/v2.0.x/example-hybrid).
+
+![poc-setup](assets/istio-apigee-envoy.png)
+
+[Apigee protected Envoy Proxies in docker containers](https://cloud.google.com/apigee/docs/api-platform/envoy-adapter/v2.0.x/example-apigee).
+
+![poc-setup](assets/standalone-apigee-envoy.png)
+
+### Installation - X/Hybrid platform
+
+1. **[Create a Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)** and connect it to an existing Billing account.
+
+1. **(For the Kubernetes environment demo) Setup a GKE cluster. Have the cluster enabled with Anthos Service Mesh. Checkout this **[gke poc toolkit](https://github.com/GoogleCloudPlatform/gke-poc-toolkit)** to standup a GKE cluster via automation with ASM enabled.**
+
+1. **This toolkit is validated in a Linux environment. Needed libraries : wget, jq.**
```

Comment:
Consider a separate "Dependencies" or "Prerequisites" section for your README to capture these. The previous point for GKE cluster with ASM installed could also be added to the same section.

Reply:
Some varying examples:
https://github.com/apigee/devrel/tree/main/tools/oas-apigee-mock#prerequisites
https://github.com/apigee/devrel/tree/main/tools/hybrid-quickstart#prerequisites
https://github.com/apigee/devrel/tree/main/references/auth-schemes#dependencies
https://github.com/apigee/devrel/tree/main/references/product-recommendations#prerequisites